### ISSUE TITLE
Problem in pre-training phase.

### ISSUE THREAD

**Issue by linjiatai** [LUSSeg/PASS#1]
Hi, I have read your paper and we are very interested in your research.

When I have run your code on the ImageNetS50, we found that the training loss of pre-training phase always become Nan.
<img width="736" alt="微信图片_20221107191103" src="https://user-images.githubusercontent.com/93721368/200298083-e909a02d-85a3-482d-8551-27154ddf3bb9.png">

I only modified the hyperparameters (N_GPU=1, base_lr=0.01, use_fp16=false) in the script file ([luss50_pass.sh](https://github.com/LUSSeg/PASS/blob/main/scripts/luss50_pass.sh)) without any other modification on this method.

**Comment by gasvn** [comment]
Please keep the parameters e.g. (lr and overall batch size) to be the same as our settings. When changing the n_GPU to other numbers,  you need to make sure that  N_GPU x bs is unchanged, e.g. when changing the n_gpu from 8 to 2, the batch size should x 4 to keep the overall batch size unchanged.
